### PR TITLE
Create de.json

### PR DESCRIPTION
**Pull request by McMornan**
Added german language translation first version

### CODE REVIEW

File: translations/de.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    "yesterday": "Gestern",
+    "today": "Heute",
+    "unknown": "unbekannte Darstellung!",
+
+  "_header": "*******************",
+  "_section": "Car status",
+  "_header": "*******************",
+    "charging_at": "{NAME} lädt auf mit",
+    "charging_at_plural": "{NAME} und {NUM} laden auf mit",
+    "consuming": "{NAME} is consuming",
```
Comment (MikeBishop) [inline translations/de.json:23]:
```suggestion
    "consuming": "{NAME} verbraucht",
```
...if I'm extrapolating from House correctly?

File: translations/de.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    "driving": "{NAME} fährt {LOCATION}",
+    "parked": "{NAME} parkt {LOCATION}",
+    "charging": "{NAME} lädt auf {LOCATION}",
+    "not_charging": "{NAME} ist hier {LOCATION} und lädt nicht",
+      "_comment": "...wobei {LOCATION} ist entweder:",
+        "at_home": "zuhause",
+        "elsewhere": "in {TOWN}",
+
+    "more_than_day": "Abgeschlossen in {DAYS}{LISTSEP}{HOURS}",
+    "less_than_day": "Abgeschlossen in {HOURS}{LISTSEP}{MINUTES}",
```
Comment (MikeBishop) [inline translations/de.json:35]:
```suggestion
    "completion_time": "Abgeschlossen in {DAYS}{LISTSEP1}{HOURS}{LISTSEP2}{MINUTES}",
```
Sorry, I changed this one out from under you.

**Review by MikeBishop — APPROVED** [review]

**Comment by McMornan** [comment]
Yes, sorry for the oversight. That’s ok.

 

Von: Mike Bishop <notifications@github.com> 
Gesendet: Donnerstag, 31. Dezember 2020 17:20
An: MikeBishop/MMM-Powerwall <MMM-Powerwall@noreply.github.com>
Cc: McMornan <deutsch.lars@gmail.com>; Author <author@noreply.github.com>
Betreff: Re: [MikeBishop/MMM-Powerwall] Create de.json (#30)

 

@MikeBishop approved this pull request.

  _____  

In translations/de.json <https://github.com/MikeBishop/MMM-Powerwall/pull/30#discussion_r550522366> :

> +    "house": "Haus",
 
+    "car": "Auto lädt",
 
+
 
+  "_header": "*******************",
 
+  "_section": "General UI terms",
 
+  "_header": "*******************",
 
+    "yesterday": "Gestern",
 
+    "today": "Heute",
 
+    "unknown": "unbekannte Darstellung!",
 
+
 
+  "_header": "*******************",
 
+  "_section": "Car status",
 
+  "_header": "*******************",
 
+    "charging_at": "{NAME} lädt auf mit",
 
+    "charging_at_plural": "{NAME} und {NUM} laden auf mit",
 
+    "consuming": "{NAME} is consuming",
 

⬇️ Suggested change 

-    "consuming": "{NAME} is consuming",
 
+    "consuming": "{NAME} verbraucht",
 

...if I'm extrapolating from House correctly?

  _____  

In translations/de.json <https://github.com/MikeBishop/MMM-Powerwall/pull/30#discussion_r550524561> :

> +    "more_than_day": "Abgeschlossen in {DAYS}{LISTSEP}{HOURS}",
 
+    "less_than_day": "Abgeschlossen in {HOURS}{LISTSEP}{MINUTES}",
 

⬇️ Suggested change 

-    "more_than_day": "Abgeschlossen in {DAYS}{LISTSEP}{HOURS}",
 
-    "less_than_day": "Abgeschlossen in {HOURS}{LISTSEP}{MINUTES}",
 
+    "completion_time": "Abgeschlossen in {DAYS}{LISTSEP1}{HOURS}{LISTSEP2}{MINUTES}",
 

Sorry, I changed this one out from under you.

**Comment by McMornan** [comment]
btw: any idea why your graphs suddenly take up the whole screen when I use the translation branch ?

**Pull request merged**